tests/manifold/transfinite_manifold_03.output

```diff
@@ -7,19 +7,19 @@ DEAL::80   cells, degree=4: 3.14159265350
 DEAL::
 DEAL::Testing with SphericalManifold dim=2, spacedim=2
 DEAL::80   cells, degree=1: 3.06146745892 
-DEAL::80   cells, degree=2: 3.14143771670 
-DEAL::80   cells, degree=3: 3.14158706712 
-DEAL::80   cells, degree=4: 3.14159263998 
+DEAL::80   cells, degree=2: 3.14064643171 
+DEAL::80   cells, degree=3: 3.14158597871 
+DEAL::80   cells, degree=4: 3.14159257389 
```

Comment:
Go down a few lines and you see that the 3D sphere has better results for cubic polynomials. It is a bit of a coincidence here, as far as I can tell, so I would not give it too much value. The difference is that the former results used the method implemented in #3886 of using all points on the vertices and lines of a `MappingQGeneric` to define the points on a quad, which I did not want to do with `TransfiniteInterpolationManifold` (after resolving the bug). Well, the Laplace smoothing should not make things worse - as the results demonstrate - but I do not really have a convincing argument of why those two should be needed together, so I do not want to pay the performance penalty of transforming more points than necessary with the new manifold (even though I spend a good deal of yesterday in speeding it up, it is still quite expensive).